tests/problems/cross_modality/test_mixins.py

```diff
@@ -0,0 +1,72 @@
+from typing import Mapping, Optional
+
+import pytest
+
+import numpy as np
+import pandas as pd
+
+from anndata import AnnData
+
+from moscot.problems.cross_modality import TranslationProblem
+from tests._utils import MockSolverOutput, _adata_modality_split
+
+
+class TestCrossModalityTranslationAnalysisMixin:
+    @pytest.mark.parametrize("src_attr", ["emb_src", {"attr": "obsm", "key": "emb_src"}])
```

Comment:
It would be nice to test for another `attr` and/or `key`.

Reply:
@nosander can we add this?